maestro-cli/src/main/java/maestro/cli/driver/DriverBuilder.kt

```diff
@@ -8,6 +8,9 @@ import java.util.concurrent.TimeUnit
 import kotlin.io.path.pathString
 
 class DriverBuilder(private val processBuilderFactory: XcodeBuildProcessBuilderFactory = XcodeBuildProcessBuilderFactory()) {
+    private val XCODEBUILD_WAIT_TIME: Long by lazy {
+        System.getenv("XCODEBUILD_WAIT_TIME")?.toLongOrNull() ?: DEFAULT_XCODEBUILD_WAIT_TIME
```

Comment:
Can you make this prefixed with MAESTRO_? That way it’s namespace like the other env vars and isn’t likely to clash with other systems